plugin/hover.py

```diff
@@ -324,11 +324,16 @@ def _on_navigate(self, href: str, point: int) -> None:
                 self.handle_code_action_select(config_name, 0)
         elif is_location_href(href):
             session_name, uri, row, col_utf16 = unpack_href_location(href)
-            session = self.session_by_name(session_name)
-            if session:
-                position = {"line": row, "character": col_utf16}  # type: Position
-                r = {"start": position, "end": position}  # type: RangeLsp
-                sublime.set_timeout_async(functools.partial(session.open_uri_async, uri, r))
+
+            if (uri.lower().startswith("http://") or uri.lower().startswith("https://")):
+                if not webbrowser.open(uri):
+                    debug("failed to open:", uri)
```

Comment:
This part can be also extracted to a "private" method(`_open_url(uri)`) in this file as it is used in the `else` branch below